src/Detection/Framework/Astro.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Utopia\Detector\Detection\Framework;
+
+use Utopia\Detector\Detection\Framework;
+
+class Astro extends Framework
+{
+    public function getName(): string
+    {
+        return 'Astro';
+    }
+
+    public function getFiles(): array
+    {
+        return ['astro.config.mjs', 'astro.config.js', '.astro/'];
```

Comment:
` '.astro/'` is not a file, doesnt fit `getFiles` method name.
If we want to do this, always look for specific file inside that is always present.

Plus, those dot folders in frameworks are temporary cache, and I don't think they are meant to be pushed to git, so we wont be seeing them anyway. Please double-check with docs of a few frameworks